nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/revision2/automation.dm

```diff
@@ -356,25 +412,46 @@
 /obj/machinery/ammo_sorter/proc/unload(atom/movable/AM)
 	if(!loaded.len)
 		return FALSE
-	playsound(src, 'nsv13/sound/effects/ship/mac_load.ogg', 100, 1)
-	flick("ammorack_dispense", src)
-	loaded -= AM
-	//Load it out the back.
-	AM.forceMove(get_turf(get_step(src, dir)))
+	weardown()
+	if(jammed)
+		playsound(src, 'nsv13/sound/effects/ship/mac_load_jam.ogg', 100, 1)
+		return
+	else
+		playsound(src, 'nsv13/sound/effects/ship/mac_load.ogg', 100, 1)
+		flick("ammorack_dispense", src)
+		loaded -= AM
+		//Load it out the back.
+		AM.forceMove(get_turf(get_step(src, dir)))
+		
 
 /obj/machinery/ammo_sorter/proc/load(atom/movable/A, mob/user)
 	if(length(loaded) >= max_capacity)
 		if(user)
 			to_chat(user, "<span class='warning'>[src] is full!</span>")
 		loading = FALSE
 		return FALSE
-	if(istype(A, /obj/item/ship_weapon/ammunition) || istype(A, /obj/item/powder_bag))
-		playsound(src, 'nsv13/sound/effects/ship/mac_load.ogg', 100, 1)
-		flick("ammorack_dispense", src)
-		A.forceMove(src)
-		loading = FALSE
-		loaded += A
-		return TRUE
+	weardown()
+	if(jammed)
+		playsound(src, 'nsv13/sound/effects/ship/mac_load_jam.ogg', 100, 1)
+		return
 	else
-		loading = FALSE
-		return FALSE
+		if(istype(A, /obj/item/ship_weapon/ammunition) || istype(A, /obj/item/powder_bag))
+			playsound(src, 'nsv13/sound/effects/ship/mac_load.ogg', 100, 1)
+			flick("ammorack_dispense", src)
+			A.forceMove(src)
+			loading = FALSE
+			loaded += A
+			return TRUE
+		else
+			loading = FALSE
+			return FALSE
+
+/obj/machinery/ammo_sorter/proc/weardown()
+	if(durability > 0) //don't go under 0, that's bad, this whole if statement handles specifically the wear&tear and it's jamming at 0% durability.
+		durability -= rand(0,3) //pulling stuff out wears it down, now randomly. Feel free to tell me how this is done correctly.
+	else 
+		jammed = TRUE // if it's at 0, just kinda jam it.
+		durability = 0 // in case an admin plays with this and doesn't know how to use it, we reset it here for good measure.
+	jamchance = ((durability - 100) * (-1)) / 4 //inverting the 100-0 durability into a 0-100 probability to jam, then halving it
+	if(prob(jamchance))
+		jammed = TRUE
```

Comment:
Line ending.

Reply:
i'm sorry, please elaborate. do you want me to add a new line at the end?

Reply:
consulting with other sources has confirmed my suspicions about this cryptic review. I have added a new line.